src/sql/CreateStatement.h

```diff
@@ -33,17 +33,42 @@ struct ColumnDefinition : TableElement {
 
   ~ColumnDefinition() override;
 
-  void setNullableExplicit() {
-    nullable = false;
+  bool trySetNullableExplicit() {
```

Comment:
I personally do not like that we have to maintain state in the loop. (The old code was easier to read for me). I would simply go over the vector multiple times if the code is in turn easier to understand: Basically: if (NULL in vector and NOT NULL in vector): error.